### DISCUSSION TITLE
List component like-ios

### DISCUSSION THREAD

**Discussion by vitalijalbu** [mantinedev/mantine#3369] [Feature requests]
Hi guys, it would be great if introducing a list component ios style,
Demo: https://framework7.io/docs/list-view

![Screenshot 2023-01-14 alle 22 33 32](https://user-images.githubusercontent.com/40239902/212497980-84617499-b3ad-45f9-9517-6edbe7ddc868.png)

**Comment by rtivital** [comment]
You can use NavLink for that – https://mantine.dev/core/nav-link/
